services/core/shared/indexdb/mysql.js

```diff
@@ -186,7 +186,7 @@ const getDbInstance = async (tableName, tableConfig, connEndpoint = config.endpo
 		.whereIn(primaryKey, ids)
 		.del();
 
-	const count = async () => knex(tableName).count({ count: 'id' });
+	const count = async () => knex(tableName).count({ count: '*' });
```

Comment:
Using a single column based count query would be more efficient. Kindly consider reverting the change.
```suggestion
	const count = async () => knex(tableName).count({ count: 'id' });
```

Reply:
Updated